packages/dds/tree/src/test/domains/json/twitter.ts

```diff
@@ -0,0 +1,487 @@
+        profile_background_image_url: "http://abs.twimg.com/images/themes/theme1/bg.png",
+        profile_background_image_url_https: "https://abs.twimg.com/images/themes/theme1/bg.png",
+        profile_background_tile: random.bool(0.5),
+        profile_image_url: "http://pbs.twimg.com/profile_images/495353473886478336/S-4B_RVl_normal.jpeg",
+        profile_image_url_https: "https://pbs.twimg.com/profile_images/495353473886478336/S-4B_RVl_normal.jpeg",
+        profile_banner_url: "https://pbs.twimg.com/profile_banners/2699365116/1406936481",
+        profile_link_color: getRandomEnglishString(random, true, 6, 6),
+        profile_sidebar_border_color: getRandomEnglishString(random, true, 6, 6),
+        profile_sidebar_fill_color: getRandomEnglishString(random, true, 6, 6),
+        profile_text_color: getRandomEnglishString(random, true, 6, 6),
+        profile_use_background_image: random.bool(0.5),
+        default_profile: random.bool(0.5),
+        default_profile_image: random.bool(0.5),
+        following: random.bool(0.5),
+        follow_request_sent: random.bool(0.5),
+        notifications: random.bool(0.5),
+    };
+    if (shouldAddUrlUrlsEntity) {
+        user.entities.url = {
+            urls: [
+                {
+                    url: "http://t.co/V4oyL0xtZk",
+                    expanded_url: "http://astore.amazon.co.jp/furniturewood-22",
+                    display_url: "astore.amazon.co.jp/furniturewood-…",
+                    indices: [
+                        random.integer(0, 199),
+                        random.integer(0, 199),
+                    ],
+                },
+            ],
+        };
+    }
+    if (shouldAddDescriptionUrlsEntity) {
+        user.entities.description.urls.push(
+            {
+                url: "http://t.co/8E91tqoeKX",
+                expanded_url: "http://ameblo.jp/2no38mae/",
+                display_url: "ameblo.jp/2no38mae/",
+                indices: [
+                    random.integer(0, 199),
+                    random.integer(0, 199),
+                ],
+            },
+        );
+    }
+
+    return user;
+}
+/* eslint-enable */
+
+// This includes common and uncommon kanji characters
+// but not rare kanji characters (3400 - 4dbf) as none were found in the source twitter json.
+function getRandomKanjiString(random = makeRandom(), minLen: number, maxLen: number) {
+    return getRandomStringByCharCode(random, minLen, maxLen, 0x4e00, 0x9faf);
+}
+
+// This is specifically formatted like the twitter json dates
+// (<3-letter-weekday> MMM DD HH:MM:SS <4-digit-TimezoneOffset> YYYY)
+function getRandomDateString(random = makeRandom(), start: Date, end: Date) {
+    const dateS = new Date(start.getTime() + random.real() * (end.getTime() - start.getTime())).toString();
+    return `${dateS.substring(0, 10)} ${dateS.substring(16, 24)} ` +
+    `${dateS.substring(28, 33)} ${dateS.substring(11, 15)}`;
```

Comment:
I believe 'Date' is coercable to/from number, so something like this would work:

```ts
const date = (min: Date, max: Date): Date => {
  return new Date(random.integer(+min, +max));
}
```

Then possibly you can get the right string format with one of the formatting functions?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#formats_of_tostring_method_return_values

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
- I looked over those and none of the formats match what twitter.json is using sadly.
- I did switch the random date logic to use what you provided in your example